test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -233,6 +244,22 @@ public String execInPod(String podName, String container, String... command) {
                 .collect(Collectors.toList());
     }
 
+    public List<Pod> listPodsByPrefixInNameWithDynamicWait(String namespaceName, String podNamePrefix) {
+        AtomicReference<List<Pod>> result = new AtomicReference<>();
+
+        TestUtils.waitFor("pod with prefix" + podNamePrefix + " is present.", Duration.ofSeconds(10).toMillis(), Duration.ofMinutes(16).toMillis(),
```

Comment:
magic numbers

Reply:
Because it's in the `test module` and I cant't add systemtest module inside the test because there will be a circular dependency between modules `api` and `mockkube`.